fb_apcupsd/README.md

```diff
@@ -0,0 +1,72 @@
+fb_apcupsd Cookbook
+====================
+This cookbook installs and configures apcupsd, the APC UPS Power Management
+daemon and its web interface.
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_apcupsd']['config']
+* node['fb_apcupsd']['hosts']
+
+Usage
+-----
+### Daemon
+Include `fb_apcupsd::default` to install apcupsd and start its service. The 
+daemon is configured via the `node['fb_apcupsd']['config']` attribute, which will
+be used to render the main configuration file at `/etc/apcupsd/apcupsd.conf`.
+Please refer to the [upstream documentation](http://www.apcupsd.org/manual/manual.html#configuration-directive-reference) for the available options. The 
```

Comment:
just mention people can see the attributes file for defaults provided by this cookbook and that they attempt to follow Debian defaults. DOn't put this in two places we have to keep up in sync.